lte/cloud/go/services/subscriberdb/load.go

```diff
@@ -63,6 +64,32 @@ func LoadSubProtosPage(
 	return subProtos, nextToken, nil
 }
 
+func SerializeSubscribers(subProtos []*lte_protos.SubscriberData) (map[string][]byte, error) {
+	subProtosSerialized := map[string][]byte{}
```

Comment:
Nit here and below: I would just call these "subsSerialized" (serialized) and subs (deserialized)